bin/pycbc_make_sky_grid

```diff
@@ -42,6 +42,92 @@ def cart_to_spher(sky_points):
     spher[spher[:, 0] <0, 0] += 2 * np.pi
     return spher
 
+def make_single_det_grid(args):
+    '''
+    Compute a trivial skygrid object with a single point 
+    '''
```

Comment:
```suggestion
    """Construct a sky grid for a single-detector analysis.
    Since a one-detector network has no sky localization capability,
    we just put a single point at the center of the input distribution.
    """
```